shopfloor_mobile_base/static/wms/src/components/screen.js

```diff
@@ -339,3 +339,14 @@ Vue.component("app-bar-actions", {
     </div>
     `,
 });
+
+Vue.component("app-version-footer", {
+    template: `
+        <v-footer absolute padless>
+                <v-col class="text-center font-weight-light" cols="12">
+                    <span class="version">{{ $t('screen.home.version') }}</span> <span class="version-number" v-text="$root.app_info.version" />
+                </v-col>
+            </v-footer>
+        <div>
```

Comment:
FTR this tmpl is broken here: this `<div />` is a left over. I'm fixing it on #469